README.md

```diff
@@ -29,23 +30,54 @@ Installs Splunk or Universal Forwarder.
-| package | `:splunk` or `:universal_forwarder` | **Yes**  |         | Specifies the Splunk package to uninstall. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end`. |
+| Name    |               Type(s)               | Required | Default | Description                                                                                                                                                                                        |
+|:--------|:-----------------------------------:|:--------:|:--------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| package | `:splunk` or `:universal_forwarder` | **Yes**  |         | Specifies the Splunk package to uninstall. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end` |
 
 
+### splunk_service
+Manages an installation of Splunk
+
+**The following actions share the same properties:**
+
+##### Action *:start*
+Starts the Splunk daemon if not already running.
+If the ulimit is changed, invokes a restart of the daemon at the end of the run.
+
+##### Action *:restart*
+Restarts the Splunk daemon, or starts it if not already running.
+
+Properties:
+
+| Name    |               Type(s)               | Required | Default                                                         | Description                                                                                                                                                                                      |
+|:--------|:-----------------------------------:|:--------:|:----------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| package | `:splunk` or `:universal_forwarder` | **Yes**  |                                                                 | Specifies the Splunk package to install. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end` |
+| user    |               String                |    No    | Owner of the specified Splunk installation, or the current user | User to run Splunk as. This is the user that will be used to run the Splunk service.                                                                                                             |
```

Comment:
I'm not sure what you're asking about exactly. It will determine which user the install resource used by the owner of the splunk directory... except for on Windows, Chef can't seem to find that information. I'm not sure if we have a directory permissions issue on Windows, though. Besides that, I also think it's possible to run Splunk as a user that has permissions to the directory but is not the owner, so I don't necessarily want to restrict it to only the determined install user.